pkg/agent/client.go

```diff
@@ -507,9 +507,10 @@ func (a *Client) remoteToProxy(connID int64, ctx *connContext) {
 			klog.V(2).InfoS("connection EOF", "connectionID", connID)
 			return
 		} else if err != nil {
+			// "use of closed network connection" errors are expected upon receiving CLOSE_REQ
+			// If connID doesn't exist in connManager, we assume the connection was meant to be closed.
 			if _, ok := a.connManager.Get(connID); !ok {
-				// Normal when receive a CLOSE_REQ
-				klog.V(5).InfoS("read from a normally closed connection", "connectionID", connID, "err", err)
+				klog.V(5).InfoS("reading to a closed connection", "connectionID", connID, "err", err)
```

Comment:
Any reason this was changed to "reading to"? I normally think "reading from" or "writing to".